research/a3c_blogpost/a3c_cartpole.py

```diff
@@ -0,0 +1,365 @@
+import os
+os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"   # see issue #152
```

Comment:
Does the device order matter if we're setting visible devices to empty?

Reply:
I was referring to https://github.com/keras-team/keras/issues/4613, but that line doesn't seem to affect anything. Removed.